staging/src/k8s.io/apiserver/pkg/server/options/audit.go

```diff
@@ -249,14 +268,32 @@ func (o *AuditWebhookOptions) AddFlags(fs *pflag.FlagSet) {
 		"Strategy for sending audit events. Blocking indicates sending events should block"+
 			" server responses. Batch causes the webhook to buffer and send events"+
 			" asynchronously. Known modes are "+strings.Join(pluginwebhook.AllowedModes, ",")+".")
+	fs.IntVar(&o.BatchConfig.BufferSize, "audit-webhook-batch-buffer-size",
+		o.BatchConfig.BufferSize, "The size of the buffer to store events before "+
+			"batching and sending to the webhook. Only used in batch mode.")
```

Comment:
The effective max buffer is actually buffer size + max batch size. Might be worth saying here? Although I think the total number of events that could be held in memory is actually unbounded (see this comment: https://github.com/kubernetes/kubernetes/pull/53105#discussion_r148090336)

Reply:
> Might be worth saying here?

Won't introducing another "buffer" in the help text cause confusion? I'm not sure how to phrase it to avoid that

> Although I think the total number of events that could be held in memory is actually unbounded

Well, not unbounded, it's bounded by `buffer size + (batch size * throttle QPS * (request timeout * number of retries + backoff time))` which is ~2 millions. Though I think apiserver will die before reaching this number